elixir/test/contracts_test.exs

```diff
@@ -40,12 +40,23 @@ defmodule Trento.ContractsTest do
       id: cloudevent_id,
       source: "wandalorian",
       spec_version: "1.0",
-      type: "Test.Event"
+      type: "Test.Event",
+      attributes: %{
+        "timestamp" =>
```

Comment:
We would need to change this to pass a known timestamp, instead of simply using seconds